mixer/pkg/attribute/envoyProtoBag.go

```diff
@@ -173,7 +173,7 @@ func AccessLogProtoBag(msg *accesslog.StreamAccessLogsMessage, num int) *EnvoyPr
 			reqMap["connection.sent.bytes"] = int64(connection.GetSentBytes())
 		}
 	}
-	reqMap["context.reporter.kind"] = "inbound"
+	reqMap["context.reporter.kind"] = "unknown"
```

Comment:
iirc, we used to have rule matches based on inbound/outbound (client/server side metrics, etc.). is it ok to not try to figure out the kind here?

Reply:
Yeah, I cannot think of a way to distinguish this. Now I think about this, it seems like it indeed bring not much value making it `unknown`. The limitation has already been documented pretty clear, so making it `inbound` might save a configuration change for users. I reverted back this change.